tsconfig.base.json

```diff
@@ -15,14 +15,17 @@
     "skipDefaultLibCheck": true,
     "baseUrl": ".",
     "paths": {
-      "@elewa-website/elements/layout/header": [
-        "libs/elements/layout/header/src/index.ts"
-      ],
```

Comment:
This removes the header Modules' aliased path from the codebase. Should still be present

Reply:
i wasn't the one who added this i thing it was the person who worked on the websites header

Reply:
but let me try to resolve it

Reply:
No, but your PR removes it. This most likely happened when resolving conflicts

Reply:
This part is Okay, the import is in line 26.